javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md

```diff
@@ -85,8 +79,7 @@ Then we can deny access to the proxy by calling the revoke method:
 ```javascript
 p.revoke();
 p.a;
-// VM181:1 Uncaught TypeError:
-// p.revoke is not a function(...)
+// TypeError: p.revoke is not a function
```

Comment:
![image](https://user-images.githubusercontent.com/19802557/77925868-4f0e7180-729d-11ea-9706-908bfd4f53ab.png)

updated the error message